llvm/include/llvm/IR/DIBuilder.h

```diff
@@ -488,6 +488,9 @@ namespace llvm {
     /// \param Elements     Struct elements.
     /// \param RunTimeLang  Optional parameter, Objective-C runtime version.
     /// \param UniqueIdentifier A unique identifier for the struct.
+    /// \param SpecificationOf The type that this type completes (is a
+    /// specification of). For example, this could be a templated type whose
+    /// template parameters have been substituted in.
```

Comment:
If you are using the word `template` you're confusing an image of C++, but, this isn't actually used for C++ at the moment. So maybe it's better to explain this in terms of what this is used for in the Swift frontend?

Reply:
Agreed. DWARF does not describe non-instantiated templates, so this explanation can only be confusing and misleading.

Reply:
Ok, I'm removing this reference to templates and adding that this is used by Swift to represent generics.